css/highcharts.css

```diff
@@ -819,7 +1053,7 @@ text.highcharts-drilldown-data-label,
 .highcharts-no-data text {
     font-weight: bold;
     font-size: 12px;
-    fill: $neutral-color-60;
+    fill: var(--highcharts-neutral-color-60);
 }
 
 /* Drag-panes module */
```

Comment:
Line no 1062 (can't comment that line directly) could also use  `--highcharts-neutral-color-100` instead of hard-coded `black`, but no strong opinion here